### DISCUSSION TITLE
Is there a way to check at runtime whether the notebook is running in jupyterlite vs locally?

### DISCUSSION THREAD

**Discussion by superlopuh** [jupyterlite/jupyterlite#945] [Q&A] [answered]
I'd like to do different setup depending on the environment, specifically something like this:

``` python
if jupyterlite:
  # Install pre-packaged hello wheel in jupyterlite
  %pip install hello
else:
  # import hello package from cloned git repo for local development
  import src
  src.path.append('..')
```

I did some googling and reading of docs and didn't find it. Help appreciated!

**Comment by bollwyvl** [comment, marked as answer]
See https://github.com/jupyterlite/jupyterlite/discussions/875

**Reply by superlopuh** [reply]
Thank you! Keyword mismatch in search, looks like exactly the answer I needed
